3600/3/app/controllers/user_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require_relative '../../helpers/user_helper'
+require 'digest'
+
+class UserController < Sinatra::Base
+  set views: proc { File.join(root, '../views/') }
```

Comment:
Move to AppController

Reply:
Moved

Reply:
Moved
![Screenshot from 2019-08-02 22-51-25](https://user-images.githubusercontent.com/46228563/62395157-1edcca80-b578-11e9-9e29-451754486bbe.png)